code/stable/projectile/SRS/Projectile_SRS.tex

```diff
@@ -350,20 +352,20 @@ \subsubsection{General Definitions}
 \end{minipage}
 \paragraph{Detailed derivation of rectilinear velocity:}
 \label{GD:rectVelDeriv}
-Assume we have rectilinear motion of a particle (of negligible size and shape, from \hyperref[pointMass]{A: pointMass}); that is, motion in a straight line. The velocity is $v$ and the acceleration is $a$. The motion in \hyperref[TM:acceleration]{TM: acceleration} is now one-dimensional with a constant acceleration, represented by ${a^{c}}$. The initial velocity (at $t=0$, from \hyperref[timeStartZero]{A: timeStartZero}) is represented by ${v^{\text{i}}}$. From \hyperref[TM:acceleration]{TM: acceleration}, using the above symbols we have:
+Assume we have rectilinear motion of a particle (of negligible size and shape, from \hyperref[pointMass]{A: pointMass}); that is, motion in a straight line. The velocity is ${v^{\text{f}}}$ and the acceleration is $a$. The motion in \hyperref[TM:acceleration]{TM: acceleration} is now one-dimensional with a constant acceleration, represented by ${a^{c}}$. The initial velocity (at $t=0$, from \hyperref[timeStartZero]{A: timeStartZero}) is represented by ${v^{\text{i}}}$. From \hyperref[TM:acceleration]{TM: acceleration}, using the above symbols we have:
```

Comment:
Here, it should definitely by $v$, which is the velocity at some time $t$, which is not necessarily the final velocity. Somewhere the wrong object is being used.

Reply:
To fix this, I had to replace the "v" symbol left for the vector speed variant. I just changed it to a bold faced v with a superscript "speed". Is there any preference for this?